internal/xds/rbac/rbac_engine.go

```diff
@@ -77,6 +77,9 @@ func (cre *ChainEngine) IsAuthorized(ctx context.Context) error {
 		logger.Errorf("newRPCData: %v", err)
 		return status.Errorf(codes.Internal, "gRPC RBAC: %v", err)
 	}
+	if logger.V(logLevel) {
+		logger.Infof("checking request: %v", newRPCData)
+	}
 	for _, engine := range cre.chainedEngines {
 		matchingPolicyName, ok := engine.findMatchingPolicy(rpcData)
 		if logger.V(logLevel) && ok {
```

Comment:
@menghanl
Considering the log here, I think the logs that I added to sdk_server_interceptors may be redundant. If we don't see this log, then we know we found no match in policy. So, I am thinking of removing sdk_server_interceptor logs.

Reply:
SG to delete the logs if they are redundant.
Maybe the best way to find out is to run some tests?

Reply:
+1